compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -366,23 +366,41 @@ OMR::ARM64::TreeEvaluator::lxorEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
 
+// Evaluate iand, ior and ixor
+static inline TR::Register *
+evaluateBooleanTypeInteger(TR::Node *node, TR::InstOpCode::Mnemonic regOp, TR::CodeGenerator *cg)
```

Comment:
I find this name confusing.  How about `evaluateBitwiseLogical` instead to keep the terminology consistent throughout the compiler?